sds_data_manager/lambda_code/SDSCode/batch_starter.py

```diff
@@ -450,16 +471,24 @@ def lambda_handler(event: dict, context):
             return
 
         # Start Batch Job execution for each instrument
-        for instrument in downstream_instruments_to_process:
-            filename = instrument["filename"]
-
-            batch_client.submit_job(
-                jobName=filename,
+        for downstream_data in downstream_instruments_to_process:
+            filename = downstream_data["filename"]
+
+            command = downstream_data["prepared_data"]
+            logger.info(f"Submitting job with this command - {command}")
+            # NOTE: The batch job name should contain only
+            # alphanumeric characters and hyphens.
+            logger.info(f"Job name: {instrument}-{level}-job")
+            response = batch_client.submit_job(
+                jobName=f"{instrument}-{level}-job",
                 jobQueue=job_queue,
                 jobDefinition=job_definition,
                 containerOverrides={
-                    "command": [instrument["prepared_data"]],
+                    "command": downstream_data["prepared_data"],
```

Comment:
No need to update here, but it is still a bit confusing to me the downstream/upstream and causes me to think every time. It almost seems like we could possibly only use one (upstream), and then call this `new_file` or something instead of downstream to really separate out the confusion with both streams.